.github/workflows/build.yml

```diff
@@ -548,6 +549,116 @@ jobs:
             --parameters @${{ env.repoName }}/test/scripts/parameters-deploy-agw.json \
             --template-file ${artifactName}/nestedtemplates/appGatewayNestedTemplate.json
 
+      - name: Query appGatewayURL of appgateway deployment
+        id: query-agwurl
+        uses: azure/CLI@v1
+        with:
+          azcliversion: ${{ env.azCliVersion }}
+          inlineScript: |
+            echo "query appgatewayURL"
+            appGatewayURL=$(az network public-ip show \
+              --resource-group $resourceGroup \
+              --name gwip \
+              --query dnsSettings.fqdn -o tsv)
+            echo "##[set-output name=appGatewayURL;]${appGatewayURL}"
+      - name: Save appGatewayURL for app deployement verification
+        id: save-appgatewayurl
+        run: |
+          echo ${{steps.query-agwurl.outputs.appGatewayURL}}
+          echo "appGatewayURL=${{steps.query-agwurl.outputs.appGatewayURL}}" >> $GITHUB_ENV
+          echo ${appGatewayURL}
```

Comment:
If the purpose of this `echo` is informational, prefix it with some explanation. Something like `appGatewayURL:`.